tests/runtime/utils/test_servicer_util.py

```diff
@@ -124,23 +127,19 @@ def test_servicer_util_is_protobuf_primitive_returns_true_for_primitive_types(
     sample_inference_service,
 ):
     """Test that is_protobuf_primitive_field is True when considering primitive types"""
+    predict_class = DataBase.get_class_for_name("SampleTaskRequest")
     assert is_protobuf_primitive_field(
-        sample_inference_service.messages.SampleTaskRequest().DESCRIPTOR.fields_by_name[
-            "int_type"
-        ]
+        predict_class().to_proto().DESCRIPTOR.fields_by_name["int_type"]
     )
 
 
-def test_servicer_util_is_protobuf_primitive_returns_false_for_custom_types(
-    sample_inference_service,
-):
+def test_servicer_util_is_protobuf_primitive_returns_false_for_custom_types():
```

Comment:
Do we no longer need `sample_inference_service` top ensure that `SampleTaskRequest` is present?

Reply:
I'm not sure why for `test_servicer_util_is_protobuf_primitive_returns_true_for_primitive_types()` it is needed else I get error ` ValueError: No data model class match for SampleTaskRequest` as you mention but I don't get that error for this test....not sure why though since they're so similar.... 🤔